### ISSUE TITLE
Is it possible update packages for using ruby 3+?

### ISSUE THREAD

**Issue by JustSnow** [grafana/pyroscope-rs#107]
Trying to integrate pyroscope for ruby 3.1.4/3.2.2 and getting error:
```
2023-05-31T16:01:06.246Z INFO  spytools::process::process_info > got symbol ruby_current_vm_ptr (0x00007f64bc3586a8) from library
thread '<unnamed>' panicked at 'Ruby version not supported yet: 3.2.2. In the meantime, we suggest trying `--force-version <prior version>`.', /work/pyroscope_backends/rbspy/src/core/ruby_version.rs:1790:14
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

**Comment by JustSnow** [comment]
https://github.com/rbspy/rbspy/issues/278

**Comment by korniltsev** [comment]
I can not confirm the problem

```
$ ruby --version
ruby 3.2.2 (2023-03-30 revision e51014f9c0) [x86_64-linux]

$ gem install pyroscope
Fetching ffi-1.15.5.gem
Fetching pyroscope-0.5.9-x86_64-linux.gem
Building native extensions. This could take a while...
Successfully installed ffi-1.15.5
Successfully installed pyroscope-0.5.9-x86_64-linux
Parsing documentation for ffi-1.15.5
Installing ri documentation for ffi-1.15.5
Parsing documentation for pyroscope-0.5.9-x86_64-linux
Installing ri documentation for pyroscope-0.5.9-x86_64-linux
Done installing documentation for ffi, pyroscope after 1 seconds
2 gems installed

A new release of RubyGems is available: 3.4.10 → 3.4.15!
Run `gem update --system 3.4.15` to update your installation.

$ ruby main.rb         

```
works file

Please share more info: pyroscope gem version if you use gem. cli version if you use cli. pyroscope configuration.
I'l close the issue for now. Feel free to reopen if the issue is still reproducible  with `pyroscope-0.5.9` gem  or `pyroscope-cli-0.2.7`
